edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Amqp/ClientConnectionHandler.cs

```diff
@@ -50,7 +50,7 @@ public Task<IDeviceListener> GetDeviceListener()
                                         // TODO: Implement plug and play for AMQP
                                         IDeviceListener dl = await this.connectionProvider.GetDeviceListenerAsync(this.identity, Option.None<string>());
                                         var deviceProxy = new DeviceProxy(this, this.identity);
-                                        dl.BindDeviceProxy(deviceProxy);
+                                        dl.BindDeviceProxy(deviceProxy, Option.None<Action>());
```

Comment:
I prefer having an interface instead of passing option of Action

Reply:
changed it to interface